Dockerfile

```diff
@@ -1,21 +1,24 @@
 # DOCKER-VERSION        1.3.2
 
-# Build UI assets
+# Build UI assets stage
 FROM node:8-alpine as assets
 RUN apk add git
 RUN npm install -g gulp-cli
 RUN mkdir -p /work
 COPY bootstrap /work/
 WORKDIR /work
-RUN npm install && npm rebuild node-sass && gulp
+RUN npm install && npm rebuild node-sass && gulp sass
+# Created asset have to be moved in a not mapped folder to be maintained
+RUN mkdir -p /assets && mv /work/css /assets
```

Comment:
Why this change? If it's to have a semantically named directory, let's just create `/assets` instead of `/work`.

I'd also prefer we keep the `css` subdirectory, as we may be processing JS in the future as well, and doing so will make this image more future-proof.

Reply:
If I use `work` that is a WORKDIR the generated assets are not copied in the other docker stage.

Reply:
I think I understand why this change exists now, but there's clearly some differences in how Docker works between MacOS and Linux. On linux, the assets still get copied into the container volume, despite mapping the application root to `/var/www`.